transaction-multi-payment/src/lib.rs

```diff
@@ -61,17 +61,17 @@ use frame_support::traits::IsSubType;
 
 use scale_info::TypeInfo;
 
-use crate::traits::{CurrencyWithdraw, PaymentWithdrawResult};
+use crate::traits::*;
```

Comment:
```suggestion
pub use crate::traits::*;
```
these should be public, otherwise other crates cannot use them